src/components/Snackbar/SnackbarContext.tsx

```diff
@@ -131,15 +131,42 @@ export const SnackbarProvider: React.FC = ({ children }) => {
       return null;
     }
 
+    const getPositionProps = (
+      position:
+        | 'top-left'
+        | 'top-middle'
+        | 'top-right'
+        | 'bottom-left'
+        | 'bottom-middle'
+        | 'bottom-right'
+        | undefined
+    ) => {
+      switch (position) {
+        case 'top-left':
+          return { top: 0, left: 6 };
+        case 'top-middle':
+          return { top: 0, left: '50%', style: { transform: 'translate(-50%, 0)' } };
+        case 'top-right':
+          return { top: 0, right: 6 };
+        case 'bottom-left':
+          return { bottom: 3, left: 6 };
+        case 'bottom-middle':
+          return { bottom: 3, left: '50%', style: { transform: 'translate(-50%, 0)' } };
+        case 'bottom-right':
+          return { bottom: 3, right: 6 };
+        default:
+          // bottom-left
+          return { bottom: 3, left: 6 };
+      }
+    };
     return ReactDOM.createPortal(
       <Flex
         position="fixed"
-        bottom={3}
-        right={6}
         flexDirection="column"
         justifyContent="center"
         alignItems="flex-end"
         zIndex={9999}
+        {...getPositionProps(transitions[0]?.item?.position)}
```

Comment:
`[question]` Do we know what happens if multiple snackbars are fired in different positions? From this, it would seem like the position is the first one.

I'm wondering if we need to support multiple positions, i.e. you could have a snackbar from the top and one from the bottom (prob not great design, but possible)

Reply:
I don't know what would happen if they were fired off from different positions. Like, I think it would be rare/weird for scenario would come up, but yeah, we sould make it explicit. Here, only snackbars of one position were coming in at a time, and I needed to access the props outside of the loop, which is why I just grabbed the first one. I'll ask Sam about that. Thanks!